test/unit/alignment/matrix/detail/alignment_trace_matrix_full_banded_test.cpp

```diff
@@ -110,3 +109,19 @@ using testing_types_inner = ::testing::Types<inner_iterator<trace_matrix_t>, inn
 INSTANTIATE_TYPED_TEST_CASE_P(banded_trace_matrix_inner_iterator,
                               iterator_fixture,
                               testing_types_inner);
+
+TEST(trace_matrix, trace_path)
+{
+    detail::alignment_trace_matrix_full_banded<trace_directions>
+        matrix{"acgt", "acgt", static_band{lower_bound{-3}, upper_bound{3}}};
+
+    EXPECT_THROW((matrix.trace_path(matrix_coordinate{row_index_type{7u}, column_index_type{4u}})),
+                 std::invalid_argument);
+
+    EXPECT_THROW((matrix.trace_path(matrix_coordinate{row_index_type{4u}, column_index_type{7u}})),
+                 std::invalid_argument);
+
+    auto path = matrix.trace_path(matrix_coordinate{row_index_type{4u}, column_index_type{4u}});
+
+    EXPECT_TRUE(path.empty());
```

Comment:
So the path stays empty because the matrix was not filled by the alignment algorithm?